docs/framework/wpf/controls/datagrid-styles-and-templates.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "DataGrid Styles and Templates"
+description: Learn about styles and templates for the Windows Presentation Foundation DataGrid control. Modify the ControlTemplate to give the control a unique appearance.
```

Comment:
@TimShererWithAquent in the future it would be good to jam the WPF acronym in the somewhere here. People don't generally search for "Windows Presentation Foundation" but they do search for "WPF"

Reply:
I've gone both ways on using acronyms in these, depending on how common they are. I think it adds one to the acrolynx score as appearing before the definition. These descriptions aren't searched by the search engine - using the acronym in the title and H1 are important for that. WPF is probably in the more common camp.

Reply:
Oh! I thought description factored into SEO. (also considering the title of the PR hehe) Is it purely just friendly description shown in results but not actually used in SEO?

Reply:
Yeah, this is what the work item says:

"Meta description help users understand the topic of a page.  Although not technically a ranking factor, meta descriptions present an opportunity to target keywords and present a clear explanation of the page."

But it also goes on to explain that search engines will pluck text from later in the page to fill in an empty description, which means we have no control over what users see.